packages/fetch-extension/src/pages-new/main/wallet-actions/index.tsx

```diff
@@ -49,8 +49,15 @@ export const WalletActions: React.FC<WalletActionsProps> = observer(
               background: "rgba(255,255,255,0.1)",
               height: "60px",
               marginBottom: "6px",
+              opacity: accountInfo.txTypeInProgress === "send" ? 0.5 : 1,
```

Comment:
Start using enums instead of hardcoding strings for all `accountInfo.txTypeInProgress` status.